### PR TITLE
Edit error demo and add minimal repo

### PR DESCRIPTION
**Pull request by liaosunny123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Add this ItemGroup in the project file (\*.csproj) of your app.
   private async Task InitializeLocalizer()
   {
       // Initialize a "Strings" folder in the executables folder.
-      StringsFolderPath = Path.Combine(Directory.GetCurrentDirectory(), "Strings");
+      var StringsFolderPath = Path.Combine(Directory.GetCurrentDirectory(), "Strings");
       StorageFolder stringsFolder = await StorageFolder.GetFolderFromPathAsync(StringsFolderPath);
```
Comment (AndrewKeepCoding) [inline README.md]: I guess this way is more consistent with other coding in this project.

```cs
StringsFolderPath stringsFolderPath  = Path.Combine(Directory.GetCurrentDirectory(), "Strings");
StorageFolder stringsFolder = await StorageFolder.GetFolderFromPathAsync(stringsFolderPath);
```

**Review by AndrewKeepCoding — COMMENTED** [review]
Thanks for the PR. I just added a comment on your changes. Can you edit that part?

**Comment by liaosunny123** [comment]
Ok, I edit it

**Comment by AndrewKeepCoding** [comment]
Thanks @liaosunny123 👍

**Pull request merged**